docs/Containers/Container-Tasks.md

```diff
@@ -0,0 +1,298 @@
+
+### **Task Metadata**
+
+A link to a container task's respective container replaces the typical
+host link.
+
+<img alt="containerized_task_metadata.png" height="400" src="../images/container_metadata.png" width="300"/>
+
+### **Container Page**
+
+The link in the task metadata sidebar takes you to the container page,
+which details the lifecycle of a container and their tasks. Like the
+host page, event logs exist charting the journey of a container task
+from initialization to termination. Task events such as container
+assignment and status changes are also recorded, as well as the clearing
+of a task from a container once it has run its course.
+
+![container_event_logs.png](../images/container_page.png)
+
+## Other Considerations
+
+### Disk Space
+
+For tasks that require a large amount of disk space, please be aware
+that container tasks may **not** be the best fit for your use case at this
+time.
+
+Each instance is provisioned with 200GB of space; however, the actual
+disk space available for each container task can vary depending on the
+number and the nature of tasks sharing the same instance. In this sense,
+while CPU and memory are isolated allocations to each container, disk
+space is a shared resource across all containers on the same host. As
+such, dedicating more memory and CPU for a container makes it less
+likely to share the instance's disk space with other containers.
+Conversely, less resource-hungry containers are more likely to share the
+instance with others, so they will likely have a smaller share of the
+available disk space.
+
+In practical terms, this means that while each instance has a maximum of
+200GB of disk space, please bear in mind that the effective disk space
+available to your container tasks might be less and fluctuate, as other
+containers take and release disk space as they get created and exit,
+respectively. While we work on a more robust solution to this notion of
+isolating disk space, we recommend that you keep your container task
+disk space usage to a maximum of 10GB.
+
+### Git Cloning Modules With SSH
+
+In a containerized environment, cloning modules via SSH is unsupported in order to prevent
+all teams from sharing the same SSH keys on the container instance hosts.
+
+Instead, modules must be cloned via OAuth over HTTPS. This means
+that if your modules' git clones rely on a host's SSH keys to clone a private
+repository in your existing workflow, the cloning method will need to be
+changed. This can be done by modifying the url format of the module's
+repo, e.g:
+
+```yaml
+modules:
+  - name: evergreen
+    repo: git@github.com:evergreen-ci/evergreen.git → https://github.com/evergreen-ci/evergreen.git
+```
+
+### Host.create
+
+Using our [host.create](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Commands#hostcreate)
+command will be unsupported for container tasks, as connecting to spawned hosts via SSH from a container will not be supported. 
```

Comment:
Users aren't really limited because of SSH, if I had to guess it's that the security groups/subnet configuration doesn't allow traffic between the container and host pools. I would leave it as "host.create is not supported for container tasks", and if people really want such a thing, they can ask for it.